dessia_common/datatools/dataset.py

```diff
@@ -677,7 +686,7 @@ def _to_samples(self, reference_path: str = '#'):
                 for row, dessia_object in enumerate(self.dessia_objects)]
 
     def _point_families(self):
-        return [PointFamily(GREY, list(range(len(self))))]
+        return [PointFamily(Color.dict_to_object('rgb(182, 225, 251)'), list(range(len(self))))]
```

Comment:
Generating an object from dict_to_object seems weird. Why can't we simply instantiate it ?

Reply:
yes we can do it like that!
done !